bentoml/utils/log.py

```diff
@@ -60,25 +63,9 @@ def get_logging_config_dict(logging_level, base_log_directory):
                 "formatter": "dev",
                 "class": "logging.handlers.RotatingFileHandler",
                 "filename": os.path.join(base_log_directory, "active.log"),
-                "maxBytes": 100 * 1000 * 1000,
+                "maxBytes": 100 * MEGABYTES,
                 "backupCount": 2,
             },
-            "prediction": {
-                "class": "logging.handlers.RotatingFileHandler",
-                "formatter": "prediction",
-                "level": "INFO",
-                "filename": os.path.join(base_log_directory, PREDICTION_LOG_FILENAME),
```

Comment:
I thought the plan was to keep the old configs according to the documentation, but removing this and L78 below, the config prediction_log_filename and feedback_log_filename will not longer be used right?

Reply:
Not sure if I understood, neither feedback and prediction handlers were referenced anywhere else. Should we still keep them around?

Reply:
They are being used for writing prediction/feedback to log files:
https://github.com/bentoml/BentoML/blob/v0.10.1/bentoml/server/api_server.py#L43
https://github.com/bentoml/BentoML/blob/v0.10.1/bentoml/service/inference_api.py#L32

Reply:
> I thought the plan was to keep the old configs according to the documentation, but removing this and L78 below, the config prediction_log_filename and feedback_log_filename will not longer be used right?

Discussed with @yubozhao, he wanted the prediction and feedback loggers not to log to file so I removed the RotatingFileHandler from the loggers by default.

> They are being used for writing prediction/feedback to log files:
> https://github.com/bentoml/BentoML/blob/v0.10.1/bentoml/server/api_server.py#L43
> https://github.com/bentoml/BentoML/blob/v0.10.1/bentoml/service/inference_api.py#L32

These lines are trying to get the feedback and prediction `loggers`. Loggers are preserved except the RotatingFileHandlers are removed. Loggers will still log to console.

Reply:
We still want to log to file but making that behavior configable, let’s discuss

Reply:
If we still like to make log to file configurable, we can add options to bentoml.cfg so users can configure through environment variables. Is this along your line of thinking?
- prediction_log_to_file_enabled
- feedback_log_to_file_enabled
- bentoml_log_to_file_enabled

Reply:
As discussed, addressed in the latest commit by adding configuration properties for enabling file logging.